flow/Tracing.h

```diff
@@ -95,9 +95,8 @@ struct Span {
 enum class TracerType {
 	DISABLED = 0,
 	LOG_FILE = 1,
-	NETWORK_ASYNC = 2,
-	NETWORK_LOSSY = 3,
-	END = 4
+	NETWORK_LOSSY = 2,
```

Comment:
Are these serialized anywhere? (I mean this is in master so protocol compatibility isn't an issue but it's still a good exercise to think about)

Reply:
Yeah this was a little painful to do :/. They aren't serialized anywhere but they are called separately on the client and server (although I think the tracing code would always be updated at the same time as these values..). I'm about to change this again to remove log file tracing from being tested in simulation, what would be a better way to go about updating the order here in the future?

Reply:
The conservative thing to do would be to only add new items, so you basically never change the meaning of "2" e.g.